docs/requirements.txt

```diff
@@ -1,6 +1,3 @@
 pygments
--e git+https://github.com/ipython/traitlets.git#egg=traitlets
--e git+https://github.com/jupyter/jupyter_core.git#egg=jupyter_core
--e git+https://github.com/jupyter/jupyter_client.git#egg=jupyter_client
```

Comment:
jupyter_client is not released yet (and this needs to keep a reference to direct dependencies)